packages/flutter_tools/lib/src/ios/cocoapods.dart

```diff
@@ -24,6 +24,9 @@ const String noCocoaPodsConsequence = '''
   Without resolving iOS dependencies with CocoaPods, plugins will not work on iOS.
   For more info, see https://flutter.io/platform-plugins''';
 
+const String unknownCocoaPodsConsequence = '''
+  Make sure that `pod --version` always return pure version like :''';
```

Comment:
Sounds a bit too imperative maybe? I assume everything will still work and we don't need to force the user to make their `pod --version` output `1.5.3` right? It's just to remove the warning? 

Maybe 
```
Flutter is unable to determined the installed CocoaPods's version. 
Ensure that the output of 'pod --version' contains only digits and . to be recognized by Flutter.
```